.github/workflows/build.yml

```diff
@@ -83,7 +83,11 @@ jobs:
         run: cd SampleApps/CarthageTest && carthage update --use-xcframeworks
       - name: Build CarthageTest
         run: xcodebuild -project 'SampleApps/CarthageTest/CarthageTest.xcodeproj' -scheme 'CarthageTest' clean build CODE_SIGNING_ALLOWED=NO
-  spm_xcode_beta:
+      - name: Run carthage build without --use-xcframeworks
+        run: |
+          ./carthage.sh build --no-skip-current
+          ./carthage.sh archive "BraintreeAmericanExpress" "BraintreeApplePay" "BraintreeCard" "BraintreeCore" "BraintreeDataCollector" "BraintreePaymentFlow" "BraintreePayPal" "BraintreeThreeDSecure" "BraintreeUnionPay" "BraintreeVenmo" "PayPalDataCollector" --output Braintree.framework.zip
```

Comment:
I'm not sure it's super valuable to test the archive on the PR CI. I think we can remove that and just test that the frameworks build.

Reply:
Opting to leave as-is for now.